mtest/Makefile

```diff
@@ -31,12 +30,26 @@ else
   SUITE_PACKAGE = $(SUITE)
 endif
 
+# If you want to change the Kubernetes version for mtest, specify this variable from command line.
+# e.g. $ make KUBERNETES_VERSION=1.14 placemat
+KUBERNETES_VERSION=1.15
+ifeq ($(KUBERNETES_VERSION),1.15)
+KUBECTL_VERSION = 1.15.3
```

Comment:
Please display `KUBECTL_VERSION` and `CKE_VERSION` during mtest to confirm
each test chooses appropriate kubectl and CKE versions.

Reply:
OK. I execute `ckecli version` and `kubectl version --short` in `mtest.

https://circleci.com/gh/cybozu-go/coil/929
```
STEP: printing CKE version
ckecli version 1.13.18

STEP: printing Kubernetes version
Client Version: v1.13.10
Server Version: v1.13.4
```